src/types/UiaTextRangeBase.cpp

```diff
@@ -354,49 +354,52 @@ IFACEMETHODIMP UiaTextRangeBase::FindAttribute(_In_ TEXTATTRIBUTEID /*textAttrib
 IFACEMETHODIMP UiaTextRangeBase::FindText(_In_ BSTR text,
                                           _In_ BOOL searchBackward,
                                           _In_ BOOL ignoreCase,
-                                          _Outptr_result_maybenull_ ITextRangeProvider** ppRetVal) noexcept
+                                          _Outptr_result_maybenull_ ITextRangeProvider** ppRetVal)

```

Comment:
this must be noexcept. if somethign returns an `HRESULT` you must never throw an exception from it. try and CATCH_RETURN.